workflows/metatdenovo.nf

```diff
@@ -114,39 +113,39 @@ include { UTILS_NFVALIDATION_PLUGIN                  } from '../subworkflows/nf-
 workflow METATDENOVO {
 
     take:
-    ch_samplesheet       // channel: path(sample_sheet.csv)
-    ch_versions          // channel: [ path(versions.yml) ]
+    ch_samplesheet // channel: samplesheet read in from --input
```

Comment:
Is it correct that `ch_versions` is not there any more? I see a commented line with versions below too.

Reply:
Yes, now it takes only ch_samplesheet and the ch_version is created in the workflow